.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateExtractor.cs

```diff
@@ -458,11 +483,25 @@ private void ExtendWithWeekdayAndYear(ref int startIndex, ref int endIndex, int
                     endIndex += matchYear.Length;
                 }
             }
+            else if (Config.CheckBothBeforeAfter)

```

Comment:
Similar to the comment above. Same elsewhere.

Reply:
Done (here and elsewhere)